### PR TITLE
Feature/new deployment envs

### PR DESCRIPTION
**Pull request by HuzaifaElahi**

### CODE REVIEW

File: docs/CONTRIBUTING.md [hunk 1]
```diff
@@ -9,6 +9,36 @@ Hi there! We're thrilled that you'd like to contribute to this project. Your hel
...
+
+`heroku create --remote staging`
+
+Note the name of the app created e.g `git.heroku.com/strong-river-216.git`, this is the Heroku staging environment name.
+
+Push to staging with the following command:
+
+`heroku push staging master`
+
+B) Use the following command to create a staging environment:
```
Comment (robvanderleek) [inline docs/CONTRIBUTING.md:26]: I think this should be changed to `production environment`

File: docs/CONTRIBUTING.md [hunk 2]
```diff
@@ -9,6 +9,36 @@ Hi there! We're thrilled that you'd like to contribute to this project. Your hel
...
 
+## Development setup & configuration
+
+After forking the repository you can deploy the project service to your own Heroku backend environments for production and staging. To be able to do so, you must first create the prerequisite environments using a Heroku account and login through the [Heroku CLI](https://devcenter.heroku.com/articles/heroku-cli) using the following steps.
+
+A) Use the following command to create a staging environment:
+
+`heroku create --remote staging`
+
+Note the name of the app created e.g `git.heroku.com/strong-river-216.git`, this is the Heroku staging environment name.
```
Comment (robvanderleek) [inline docs/CONTRIBUTING.md:20]: It's semantics and subjective but the thing you're referring to here is the Heroku git repository for the staging app/environment. I'm not 100% sure about Heroku terminology but I think `strong-river-216` is the app name and `git.heroku.com/strong-river-216.git` the repository name.

File: docs/CONTRIBUTING.md [hunk 3]
```diff
@@ -9,6 +9,36 @@ Hi there! We're thrilled that you'd like to contribute to this project. Your hel
...
+
+Push to staging with the following command:
+
+`heroku push staging master`
+
+B) Use the following command to create a staging environment:
+
+`heroku create --remote production`
+
+Note the name of the app created e.g `git.heroku.com/fierce-ice-327.git`, this is the Heroku Production environment name.
```
Comment (robvanderleek) [inline docs/CONTRIBUTING.md:30]: See comment for staging.

**Review by robvanderleek — COMMENTED** [review]
Hi @HuzaifaElahi 
I like this setup :)
There are a couple of minor issues in the documentation, can you have a quick look?

**Comment by HuzaifaElahi** [comment]
@robvanderleek made the changes to the doc

**Pull request merged**